articles/introduction-to-neural-networks/index.md

```diff
@@ -0,0 +1,208 @@
+
+The input layer computes the weighted input for every node. The input layer computes the weighted input for every node. The activation function is pertained to get the result as output.
+
+![Perceptron](/engineering-education/introduction-to-neural-networks/perceptron.png)
+
+[Image source](https://missinglink.ai/wp-content/uploads/2018/11/Frame-3.png)
+
+#### Feed Forward Neural Network
+
+In a feed-forward network, data moves in a single direction. It enters via input nodes and leaves through output. This is a front propagation wave. By data moving in one direction, there is no [backpropagation](https://www.guru99.com/backpropogation-neural-network.html). The backpropagation algorithm calculates the gradient of the loss function with consideration to weights in the network.
+
+The input product sum and their weights are computed. The data later is transferred to the output.
+
+Feed-forward neural networks applications are:
+
+- Speech Recognition
+- Facial Recognition
+
+![Feed forward Network](/engineering-education/introduction-to-neural-networks/fnn.png)
+
+[Image source](https://deepai.org/machine-learning-glossary-and-terms/feed-forward-neural-network)
+
+#### Radial Basis Function Neural Network
+
+[Radial Basis Function Neural Network (RBF](https://www.sciencedirect.com/topics/engineering/radial-base-function)) comprises of three layers:
+
+- [Input Layer](https://stackoverflow.com/questions/32514502/neural-networks-what-does-the-input-layer-consist-of)
+- [Hidden Layer](https://www.sciencedirect.com/topics/engineering/hidden-layer-neuron)
+- [Output Layer](https://www.scan2cad.com/user-manual/the-output-layer/)
+
+[RBF networks](https://www.sciencedirect.com/topics/chemical-engineering/radial-basis-function-networks) classify data based on the distance of any centered point and interpolation. Interpolation resizes images. Classification is executed by estimating the inputs data where each neuron reserve the data. RBF network looks for similar data points and groups them.
+
+According to [Dr. Saed Sayad](https://www.saedsayad.com/artificial_neural_network_rbf.htm), the sum and weights of hidden layer output sent to the output layer form a network of outputs. RBF network is mostly used in the power restoration system. RBF neural network reduces the power shortage.
+
+![Radial Basis Function Network](/engineering-education/introduction-to-neural-networks/rbn.png)
+
+[Image source](https://miro.medium.com/max/500/0*Dezf_up8pNrdsl34.png)
+
+#### Recurrent Neural Network
+
+In neural networks such as in a feed-forward network, data moves in one direction. The network has a shortcoming of not remembering the data in past inputs. This is where RNN comes in.
+[A Recurrent Neural Network(RNN](https://www.sciencedirect.com/topics/engineering/recurrent-neural-network)) is a network whereby the hidden layer's output is saved from being used as input. Saving the output helps with future predictions.
```

Comment:
👍  Thank you for explaining what a RNN is - I think this is a very important part to understand - if you could elaborate a bit on this sentence: 
"whereby the hidden layer's output is saved from being used as input."

Because as I'm reading it right now - it seems to say that the output that goes into a RNN hidden layer - is saved from being used as input to the next layer? 

Not sure if that is what it does or what was meant.  🤔

Reply:
Same with this comment - it was not addressed